tests/run_example.rs

```diff
@@ -0,0 +1,39 @@
+#[test]
+fn run_example() {
+    let output = std::process::Command::new("cargo")
+        .args(["nextest", "run", "--test=example", "--color=never"])
+        .env("DATATEST_FULL_SCAN_FORBIDDEN", "1")
+        .output()
+        .expect("Failed to run `cargo nextest`");
+
+    // It's a pain to make assertions on byte slices (even a subslice check isn't easy)
+    // and it's also difficult to print nice error messages. So we just assume all
+    // nextest output will be utf8 and convert it.
+    let stderr = std::str::from_utf8(&output.stderr).expect("Failed to convert stderr to utf-8");
```

Comment:
`expect` should have positive success messages, not negative ones. e.g. `"converted stderr to utf-8"`

Reply:
This is documented here: https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style